### ISSUE TITLE
Python 3 incompatible get_nndc_data.py 

### ISSUE THREAD

**Issue by scopatz** [mit-crpg/openmc#388]
Helllo all, I am seeing the following error with get nndc.  I think you just need to make the file open mode `'rb'`

```python
scopatz@nukestar01 ~/openmc/data master BA$H python get_nndc_data.py 
Downloading ENDF-B-VII.1-neutron-293.6K.tar.gz...  501910959  [100.00%]
Downloading ENDF-B-VII.1-tsl.tar.gz...   17925081  [100.00%]
Verifying MD5 checksums...
Traceback (most recent call last):
  File "get_nndc_data.py", line 76, in <module>
    downloadsum = hashlib.md5(open(f, 'r').read()).hexdigest()
  File "/home/scopatz/miniconda3/lib/python3.4/codecs.py", line 319, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte
```

**Comment by wbinventor** [comment]
This was fixed by #365 on the develop branch, but hasn't yet been released on master.

**Comment by scopatz** [comment]
Ok.  Cool, I'll just switch to develop then for this.
